tools/ci_build/github/linux/run_dockerbuild.sh

```diff
@@ -135,6 +139,10 @@ else
                 --dockerfile $DOCKER_FILE --context .
 fi
 
+if [[ $NEED_BUILD_SHARED_LIB = true ]]; then
```

Comment:
```suggestion
# Current build script doesn't support building onnxruntime with Python dependency. To enable building with PythonOp,
# we need to avoid `--no-undefined` via disabling `--build_shared_lib`. Otherwise, CIs will report `missing symbols`.
if [[ $NEED_BUILD_SHARED_LIB = true ]]; then
```

Reply:
Fair. will refine the comment and make a change.